android/build.gradle

```diff
@@ -1,4 +1,4 @@
-group 'com.example.clevertap_directcall_flutter'
+group 'com.example.clevertap_signedcall_flutter'
 version '1.0-SNAPSHOT'
```

Comment:
@shivamsharma2710 Version should be 1.0.0

Reply:
This is resolved!